typewriter/fixes/tests/test_util.py

```diff
@@ -0,0 +1,114 @@
+""" Test suite for the code in fixer_utils """
+
+# Local imports
+from lib2to3.fixer_util import (Attr, Call, Comma, Name, find_binding,
+                                is_import, syms)
+from lib2to3.pgen2 import token
+from lib2to3.pytree import Leaf, Node
+from lib2to3.tests.support import TestCase
+from lib2to3.tests.test_util import parse
+from typing import NamedTuple
+
+from .. import fixer_utils
+
+ImportInfo = NamedTuple("ImportInfo", [
+    # The bound name, i.e. "import x as y" would bind the name y for module x
+    ("binding_name", str),
+    ("imp_name", str),
+    ("node", Node)
+])
+
+
+class Test_get_import_binding_info(TestCase):
```

Comment:
Do you think we need to test _get_import_binding_info separately? It's basically just a more narrow case of `find_import_info` - AFAICT behavior should essentially be identical for code with only one import statement (or the import statement is the first one).  Also, `_get_import_binding_info` is a private function... and while I'm not rigidly against ever testing private functions, it does make it less necessary.

Is the test dataset identical between this and `Test_find_import_info`? (At least, before my suggested additions?)

Reply:
yeah we should delete the `_get_import_binding_info`